docs/02-StewardForValidators.md

```diff
@@ -0,0 +1,36 @@
+# Validators Instructions for Setting Up Steward
+
+Steward is an application intended for developers and validators on the Sommelier network.
```

Comment:
An application... that does what? Linking to a doc that explains what Steward does at a high level, and how it fits into the broader ecosystem of things Sommelier validators need to run or worry about, could be helpful here.